pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java

```diff
@@ -444,6 +444,24 @@ public synchronized List<PinotTaskConfig> getTaskConfigs(String taskName) {
     return taskConfigs;
   }
 
+  /**
+   * Get the task runtime config for the given task name. A task can have multiple subtasks, whose configs can be
+   * retrieved via the getSubtaskConfigs() method instead.
+   *
+   * @param taskName Task name
+   * @return Configs for the task returned as a Map.
+   */
+  public synchronized Map<String, String> getTaskRuntimeConfig(String taskName) {
+    JobConfig jobConfig = _taskDriver.getJobConfig(getHelixJobName(taskName));
+    jobConfig.getNumConcurrentTasksPerInstance();
+    HashMap<String, String> configs = new HashMap<>();
+    configs.put("ConcurrentTasksPerWorker", String.valueOf(jobConfig.getNumConcurrentTasksPerInstance()));
+    configs.put("TaskTimeoutInMs", String.valueOf(jobConfig.getTimeoutPerTask()));
+    configs.put("TaskExpireTimeInMs", String.valueOf(jobConfig.getExpiry()));
```

Comment:
(minor) We usually not put `In`
```suggestion
    configs.put("TaskTimeoutMs", String.valueOf(jobConfig.getTimeoutPerTask()));
    configs.put("TaskExpireTimeMs", String.valueOf(jobConfig.getExpiry()));
```